device/device.go

```diff
@@ -580,8 +440,8 @@ func deleteHost(fileName, host string) error {
 		}
 	}
 	output := strings.Join(result, "\n")
-	err = ioutil.WriteFile(fileName, []byte(output), 0644)
-	if err != nil {
+
+	if err = ioutil.WriteFile(fileName, []byte(output), 0644); err != nil {
```

Comment:
__[gometalinter]__
* [gosimple] 'if err != nil { return err }; return nil' can be simplified to 'return err'

<!-- Comment by [SideCI](https://sideci.com) -->